### PR TITLE
#171180313 Improve existing trips and bookings design

### PR DESCRIPTION
**Pull request by NiyongaboEric**
#### What does this PR do?
The PR helps to improve the existing design and functionality

#### Description of Task to be completed?
- style bookings pages
- style  trips requests

#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-backend.git`
- `cd team-odd-bn-backend`
- git checkout -b ch-display-info-171180313 
- `npm install`
- Make sure your `.env` file is updated.
- Drop existing migration `npm run dropTables`
- Update migration `npm run db-migrate`
- Populate the seed data `npm run db-seed-dev`
- `npm run dev-start` check API in development 
>View all bookings`{{url}}/api/v1/bookings`.  // try it when your bookings is empty

>view specific Trips `{{url}}/api/v1/trips/oneway`. // You should first create a trip request and try to view it

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#171180313](https://www.pivotaltracker.com/story/show/171180313)

#### Screenshots (if appropriate)

Visit bookings without creating it
Screen Shot 2020-02-10 at 17 50 32

View specific trip request
Screen Shot 2020-02-10 at 15 00 22

#### Questions:

### CODE REVIEW

**Comment by kipropbrian** [comment]
Can you move the comments down, instead of on the side.

File: src/assets/css/bookings/userBookings.scss [hunk 1]
```diff
@@ -246,3 +245,39 @@
...
+@media screen and (min-width: 760px) and (max-width: 900px) {
+  .empty-bookings-container {
+    width: 60%;
+  }
+}
+@media (min-width: 900px) {
+  .empty-bookings-container {
+    width: 50%;
+  }
+}
```
Comment (hezronkimutai) [inline src/assets/css/bookings/userBookings.scss]: eol

**Review by hezronkimutai — CHANGES_REQUESTED** [review]
Nice work @NiyongaboEric , I have left a comment, please look at it.

**Review by hezronkimutai — APPROVED** [review]

**Review by nziokaivy — APPROVED** [review]
Looks good to me! Great job @NiyoEric

**Review by victkarangwa — APPROVED** [review]
Good job @NiyoEric

File: src/assets/css/accommodation/allAccommodation.scss [hunk 2]
```diff
@@ -249,3 +248,8 @@ a.accommodation-item-container img {
 a.accommodation-item-container img:hover {
   box-shadow: 0 0 2px 1px rgba(0, 140, 186, 0.5);
 }
+
+.specific-accommodation-container{
+  height: 90%;
+  overflow-y: auto;
+}
```
Comment (william000000) [inline src/assets/css/accommodation/allAccommodation.scss:255]: End of line

**Review by william000000 — CHANGES_REQUESTED** [review]

**Review by william000000 — COMMENTED** [review]
Nice Work @NiyongaboEric

**Pull request merged**